src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,58 @@
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo,
+  isProcessed: boolean,
+  onDelete?: React.MouseEventHandler<HTMLButtonElement>,
```

Comment:
`onDelete` has the wrong type. change to:
```suggestion
  onDelete?: (id: number) => void,
```